pints/plot.py

```diff
@@ -565,7 +566,10 @@ def pairwise(samples,
         is the number of parameters.
     ``kde``
         (Optional) Set to ``True`` to use kernel-density estimation for the
-        histograms and scatter plots.
+        histograms and scatter plots. Cannot use together with ``heatmap``.
+    ``heatmap``
+        (Optional) Set to ``True`` to plot heatmap for the pairwise plots.
+        Cannot use together with ``kde``.
     ``opacity``
```

Comment:
Does opacity do anything for the heatmap?

Reply:
No, it should only affect the scatter plot -- neither kde or heatmap uses it.

Reply:
Please mention this in the docstring then!